apps/judicial-system/scheduler/src/app/app.service.ts

```
@@ -11,6 +15,46 @@ import { NotificationDispatchType } from '@island.is/judicial-system/types'
 import { appModuleConfig } from './app.config'
 import { now } from './date.factory'
 
+enum JobScheduleType {
+  EveryDayAt2 = 'everyDayAt2',
+  WeekdaysAt9 = 'weekdaysAt9',
+}
+
+type JobConfig = {
+  jobScheduleType: JobScheduleType
+  function: () => Promise<void | Logger>
```

Biome (1.9.4)

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)